src/jit/morph.cpp

```diff
@@ -11834,7 +11834,11 @@ GenTree* Compiler::fgMorphSmpOp(GenTree* tree, MorphAddrContext* mac)
             }
 
 #if USE_HELPERS_FOR_INT_DIV
-            if (typ == TYP_INT && !fgIsSignedDivOptimizable(op2))
+            if (typ == TYP_INT
+#if defined(LEGACY_BACKEND)
+                && !fgIsSignedDivOptimizable(op2)
```

Comment:
What system do we have that uses HELPERS for INT_DIV but is not a LEGACY_BACKEND?
RyuJIT Arm32 I would guess.

Reply:
Yes, `USE_HELPERS_FOR_INT_DIV` is defined only for arm.